rules/windows/process_creation/proc_creation_win_susp_guid_task_name.yml

```diff
@@ -0,0 +1,30 @@
+title: Suspicious Scheduled Task Name As GUID
+id: ff2fff64-4cd6-4a2b-ba7d-e28a30bbe66b
+status: experimental
+description: Detects creation of a scheduled task with a GUID like name
+references:
+    - https://thedfirreport.com/2022/10/31/follina-exploit-leads-to-domain-compromise/
+    - https://thedfirreport.com/2022/02/21/qbot-and-zerologon-lead-to-full-domain-compromise/
+author: Nasreddine Bencherchali
+date: 2022/10/31
+tags:
+    - attack.execution
+    - attack.t1053.005
+logsource:
+    product: windows
+    category: process_creation
+detection:
+    selection_img:
+        Image|endswith: '\schtasks.exe'
+        CommandLine|contains|all:
+            - '/Create '
+            - '}"' # Ending of the name to avoid possible FP in the rest of the commandline
```

Comment:
selection_tn have simple and double quote but the end is double quote only.
So `/Create /TN '{guid}'` may not be cauth

Do you think windows CommandLine is quite permissive for a mix like  `/Create /TN "{guid}'` ?

Reply:
You're right. It should have the other cases. I have pushed an update.